pkg/networkutils/network.go

```diff
@@ -207,9 +207,10 @@ func (n *linuxNetwork) SetupHostNetwork(vpcCIDR *net.IPNet, vpcCIDRs []*string,
 		return errors.Wrapf(err, "host network setup: failed to delete old host rule")
 	}
 
+	primaryIntf := "eth0"
```

Comment:
It's not optimal to define the variable here, then use it 144 lines later, but I also don't want to call `findPrimaryInterfaceName()` again, so I guess this is the best option.

Btw, would you mind updating the error message on line 233, it should use the value of `primaryIntf` and not be hardcoded to "eth0", that might be misleading.

Reply:
Sure, that makes sense. Thank you!